### DISCUSSION TITLE
Do order of RDA axes correspond to explained variance of explanatory variables?

### DISCUSSION THREAD

**Discussion by aguilart** [vegandevs/vegan#687] [Q&A]
Hello, I am doing an RDA where my matrix of response variables (morphological traits of individuals) is modeled against two explanatory variables and their interaction. Something like: traits ~ time * treatment. When I look at the variance explained by the factor I get the following order: time explains more, then treatment and last the interaction. As I get three RDA axes, am I right to assume that the first RDA axes correspond to time; the second to treatment and the third to the interaction?

Is this a fair way to interpret the axes?

Thanks!

CA

**Comment by jarioksa** [comment]
No, you cannot assume that. However, you also get the biplot arrows. These will show the direction of optimal separation (or correlation) of the factor. If a biplot arrow is parallel to an axis, the variable correspond to that axis. Usually they are not parallel, but oblique and then you cannot have such an interpretation. However, if you project the arrow tip to an axis, you see in which proportions the axes correspond to each variable.

**Reply by aguilart** [reply]
thanks that makes a lot of sense! I will look at the angle then of the biplot arrows to the RDA axes for interpretation
